app/resources/forms.js

```diff
@@ -367,3 +367,13 @@ function make_form_expandable(expandedSection) {
     return false;
   });
 }
+
+function form_confirmation() {
```

Comment:
It's usually preferred to use a verb for function names. e.g., confirm_submit, ask_for_form_confirmation

Also, add a short comment to explain what this function does.